SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -267,7 +270,7 @@ Interest UnrealServerInterestFactory::CreateInterest(AActor* InActor, const FCla
 	{
 		if (GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
 		{
-			AddClientInterestEntityIdQuery(ResultInterest, InActor);
+			// AddClientInterestEntityIdQuery(InActor);
 		}
 		else
```

Comment:
Just delete and making into
`if (!GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)`
?

Reply:
I did want to keep to previous method interest component method you implemented available, in case we needed to switch back (I didn't change or delete AddClientInterestEntityIdQuery for instance) - but perhaps we just remove it fully.